File: realworld-back/realworld-common/src/main/java/com/realworld/properties/ExcludePathProperties.java
package com.realworld.properties;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;
import java.util.List;

/**
 * @author YYJYP
 */
@Data
@ConfigurationProperties(prefix = "realworld.back.exclude.path")
public class ExcludePathProperties {
	private List<String> pathList;
}

File: backend-springboot/src/main/kotlin/com/yyjy/common/ExcludePathProperties.kt
package com.yyjy.common

import org.springframework.boot.context.properties.ConfigurationProperties

/**
 * @author Nyxcirea
 * @date 2026/3/2
 * @description: TODO
 */

@ConfigurationProperties("demo.exclude")
data class ExcludePathProperties(
    val paths: List<String>
)
